utils/zerovec/src/ule/mod.rs

```diff
@@ -46,6 +46,16 @@ where
     /// The error that occurs if a byte array is not valid for this ULE.
     type Error;
 
+    /// Validates a byte slice, `&[u8]`.
+    ///
+    /// If `Self` is not well-defined for all possible bit values, the bytes should be validated,
```

Comment:
"Should be validated that they can be transmuted to a `Self`"

also the description of the trait invariants should be updated on the docs above given that this function is the important one